ChipBrowne/bit105
# Team c

## Summary
Attendance records created for Joshua Favre and Dwight Harris

## Issue Being Resolved
#80 

## Testing Instructions
Verify list and anchor tag positions

*Tag Code Reviewers using the GitHub Reviewers option. (upper right)*
To Code Review successfully, leave *NO comments*, click *Approve* and then click *Submit*.
@shutdoor
